### ISSUE TITLE
Support multiple config blocks for an interface on debian

### ISSUE THREAD

**Issue by jmccann** [target/network_interfaces_v2-cookbook#20]
Example:

```
iface eth0 inet6 static
        address FE80::0202:B3FF:FE1E:8329
        netmask 64
        gateway FE80::0202:B3FF:FE1E:0
        dns-nameservers 2001:4860:4860::8888

iface eth0 inet static
        address 1.2.3.4
        netmask 255.255.255.255
        gateway 1.2.3.1
        dns-nameservers 8.8.8.8
```

**Comment by jmccann** [comment]
@martinisoft Is this supported only for ipv4 and ipv6 or can you have any number of config blocks for a physical interface ... for example would the following be valid for 2 ipv4 IPs?

```
iface eth0 inet static
        address 1.2.3.4
        netmask 255.255.255.0
        gateway 1.2.3.1
        dns-nameservers 8.8.8.8

iface eth0 inet static
        address 2.3.4.5
        netmask 255.255.255.0
        gateway 2.3.4.1
        dns-nameservers 8.8.8.8
```

**Comment by martinisoft** [comment]
@jmccann According to [Debian's manual pages](https://wiki.debian.org/NetworkConfiguration#Multiple_IP_addresses_on_one_Interface) that method is preferred for newer setups. The older `device:virtual` method seems to be on it's way out now.

**Comment by jmccann** [comment]
One thought I had to address this is to have debian config file names created by the `new_resource.name` instead of `new_resource.device`.  This would allow you to set `device` the same on multiple provider calls but generate separate config files for each.

```ruby
network_interface 'eth0_123' do
  device 'eth0'
  address '1.2.3.4'
  netmask '255.255.255.0'
  gateway '1.2.3.1'
end

network_interface 'eth0_234' do
  device 'eth0'
  address '2.3.4.5'
  netmask '255.255.255.0'
  gateway '2.3.4.1'
end
```

Would create config files ...

`/etc/network/interfaces.d/eth0_123`:
```
iface eth0 inet static
        address 1.2.3.4
        netmask 255.255.255.0
        gateway 1.2.3.1
```

`/etc/network/interfaces.d/eth0_234`:
```
iface eth0 inet static
        address 2.3.4.5
        netmask 255.255.255.0
        gateway 2.3.4.1
```

Do you think that would work @martinisoft ?

**Comment by martinisoft** [comment]
@jmccann That is actually a really nice solution. Do you want me to create a PR for it?

**Comment by jmccann** [comment]
@martinisoft That'd be awesome!  Thanks!

**Comment by jmccann** [comment]
Addressed by #26
